packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -299,6 +299,48 @@ describe('TagGroup', function () {
     expect(onRemoveSpy).toHaveBeenCalledWith('1');
   });
 
+  it('maxRows should limit the number of tags shown', function () {
+    let offsetWidth = jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect')
```

Comment:
I assume these values were lifted from the browser? they are a bit hard to reason through as a human. I can't tell if they check for any edge cases. This could be easier if whole numbers were chose as well as rounded to numbers like 5 or 10 with specific instances where we don't do that to show we are testing an edge case.

Reply:
Good point, updated